docs/data/data-grid/export/export.md

```diff
@@ -1,6 +1,6 @@
 # Data Grid - Export
 
-<p class="description">Easily export the rows in various file formats such as CSV, Excel, or PDF.</p>
+<p class="description">Easily export the rows in various file formats such as CSV, Excel, or Print.</p>
```

Comment:
I think this sentence should be totally rephrased because it doesn't make sense to describe Print as a file format. Something along the lines of (:
```suggestion
<p class="description">Easily export rows in CSV or Excel format, or open the browser's print dialog with a print-friendly view of the data grid.</p>
```

I don't love the wording, perhaps @mapache-salvaje can think of something nicer.

Reply:
Sure, fair enough 👍 

```suggestion
<p class="description">Export the rows in CSV or Excel formats or use the browser's print dialog to print or save as PDF.</p>
```

Reply:
I'm happy with @oliviertassinari 's suggestion above 😁